src/main/resources/db/migration/R__TypeCodes.sql

```diff
@@ -74,6 +74,7 @@ VALUES (1, 'User Added to Organization', 1),
        (2, 'Facility Idle', 2),
        (3, 'Facility Alert Requested', 3),
        (6, 'Accession Scheduled to End Drying', 1),
+       (7, 'Nursery Seedling Batch Ready', 1),
```

Comment:
I'm a little uneasy about reusing type IDs that used to exist but were deleted. Seems like this would run the risk of inadvertently matching existing values from before we removed an ID from this list. In theory we will always be careful to have a migration with `DELETE FROM notifications WHERE notification_type_id = 7` when we remove ID 7 from a list of type codes, but maybe we'll forget.

What do you think of using 15 instead of 7 for this?

Reply:
Ah, I didn't realize that the gaps were due to deleted types. I will definitely change this!